rf_tests.go

```diff
@@ -342,13 +342,20 @@ func deleteRFML(c cliContext) error {
 	return nil
 }
 
-// uploadRFML is a wrapper around test creating/updating functions
-func uploadRFML(c cliContext, api rfmlAPI) error {
+// uploadTests is a wrapper around test creating/updating functions
+func uploadRFTests(c cliContext, api rfAPI) error {
 	if c.Bool("synchronous-upload") {
 		rfmlUploadConcurrency = 1
 	}
 	if path := c.Args().First(); path != "" {
-		err := uploadSingleRFMLFile(path)
+		var err error
+
+		if strings.HasSuffix(path, ".rfml") {
+			err = uploadSingleRFMLFile(path)
+		} else {
+			err = uploadSingleWispFile(path)
```

Comment:
I'm aware this isn't your naming convention, and we might not want to make this change as part of this PR - but including the word `Single` in the function name is completely redundant?